pkg/security/resolvers/process/resolver_ebpf.go

```diff
@@ -493,6 +493,11 @@ func (p *EBPFResolver) retrieveExecFileFields(procExecPath string) (*model.FileF
 	return &fileFields, nil
 }
 
+// retrieveExecFileFields fetches inode metadata from kernel space
+func (p *EBPFResolver) retrieveExecFileFields(procExecPath string) (*model.FileFields, error) {
+	return p.retrieveFileFields(procExecPath)
+}
```

Comment:
Not sure to see why this change is needed

Reply:
It's not anymore, indeed